### ISSUE TITLE
Wipe migrations

### ISSUE THREAD

**Issue by jleclanche** [translate/pootle#3311]
Make sure migrations work for 2.6.0 and are synced with core then wipe them out and start fresh for core.

**Comment by unho** [comment]
@jleclanche I find this issue description confusing. Can you rephrase it?

**Comment by jleclanche** [comment]
What's unclear? We need to wipe the migrations on core before releasing 2.7.

**Comment by unho** [comment]
@jleclanche Did you mean "Make sure 2.6.0 are synced with core and work"? Which migrations are going to be wiped on which branch?

**Comment by julen** [comment]
master resets its migrations, whatever migrations existed there are ported to 2.6, 2.7 starts clean.

**Comment by unho** [comment]
@jleclanche I think this is done in 6c6d0fa1 and some other post commits like ff8aa19f.

**Comment by jleclanche** [comment]
Yes, but we'll have to do it again if there's any new migrations before then.

**Comment by unho** [comment]
@jleclanche I don't think it is necessary. Now we only have a few migrations. Also this disrupts @translate/evernote deployment.

**Comment by jleclanche** [comment]
The whole point is to get rid of *all* migrations to remove a step from the upgrade process.

**Comment by unho** [comment]
@jleclanche But @dwaynebailey wants the upgrade docs to be the same across versions, if possible. So the migrate step is going to be there, yes or yes. So I think it is no longer necessary to wipe the migrations again since now we have a small set of them.

**Comment by unho** [comment]
PR #3823 mostly fixed this issue.

There are some migrations that seem not to be possible to be squashed to just a `0001_initial` migration in `accounts` and `pootle_translationproject` apps. These can't be squashed since they depend on migrations on other apps that depend on their `0001_initial` migrations, for example `tp.0002` depends on `store.0001` that depends on `tp.0001`.

**Comment by unho** [comment]
> PR #3823 mostly fixed this issue.

> There are some migrations that seem not to be possible to be squashed to just a `0001_initial` migration in `accounts` and `pootle_translationproject` apps. These can't be squashed since they depend on migrations on other apps that depend on their `0001_initial` migrations, for example `tp.0002` depends on `store.0001` that depends on `tp.0001`.

I don't see how this can be addressed, so I am closing this as resolved.
